example4.py

```diff
@@ -1,24 +1,29 @@
-# Corrected Python code
-
 from math import pi
 
-def calculate_area(radius):
+def calculate_area(radius: float) -> float:
+    if radius < 0:
```

Comment:
Good job on utilizing an f-string for better readability and fixing the string concatenation.